static/spoofedFilesFromServer/fromSinopiaServer/resourceTemplates/MonographInstance.json

```diff
@@ -175,7 +175,7 @@
         },
```

Comment:
This is not specific to this code, but I realized another source we should use for determining such variables as `repeatable` and `mandatory` is the Library of Congress Bibframe documentation at https://www.loc.gov/bibframe/. For example, in this BIBFRAME file the 'instanceOf` propertyTemplate has false for repeatable, but I as understand BIBFRAME 2.0 ontology, at this level of modeling, Instance should have only one Work, while Works have multiple Instances.